packages/firestore/src/remote/serializer.ts

```diff
@@ -852,6 +853,68 @@ export function toQueryTarget(
   return result;
 }
 
+export function toRunAggregationQueryRequest(
+  serializer: JsonProtoSerializer,
+  target: Target
+): ProtoRunAggregationQueryRequest {
+  // Dissect the path into parent, collectionId, and optional key filter.
+  const result: ProtoRunAggregationQueryRequest = {
+    structuredAggregationQuery: {
+      aggregations: [
+        //TODO: dynamically add the aggregate fields in future development
```

Comment:
Remove this "TODO". It should be "obvious" in the future and is just a bit confusing at this point.